regulation-worker/internal/service/service.go

```diff
@@ -68,8 +68,8 @@ func (js *JobSvc) JobSvc(ctx context.Context) error {
 	}
 
 	stats.Default.NewTaggedStat("regulation_worker_deletion_time", stats.TimerType, stats.Tags{"workspaceId": job.WorkspaceID, "destinationid": destDetail.DestinationID, "destinationType": destDetail.Name, "status": string(jobStatus.Status)}).Since(deletionStart)
-	if jobStatus.Status == model.JobStatusComplete {
-		stats.Default.NewTaggedStat("regulation_worker_deleted_user_count", stats.CountType, stats.Tags{"workspaceId": job.WorkspaceID, "destinationid": destDetail.DestinationID, "destinationType": destDetail.Name}).Count(len(job.Users))
+	if jobStatus.Status == model.JobStatusComplete || jobStatus.Status == model.JobStatusAborted {
```

Comment:
why not failed as well?

Reply:
I have added new metric which covers all statuses